rpcs3/Emu/Cell/SPUDisAsm.cpp

```diff
@@ -106,6 +147,70 @@ std::pair<bool, v128> SPUDisAsm::try_get_const_value(u32 reg, u32 pc) const
 	return {};
 }
 
+std::tuple<u32, u32, u32> SPUDisAsm::test_insertation_mask(v128 mask)
```

Comment:
Just a nitpick. The problem with tuple - 3 unnamed u32 values. How am supposed to know which one which what?

Reply:
Fixed.